examples/gke/nodeconfig-alpha.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: scylla.scylladb.com/v1alpha1
+kind: NodeConfig
+metadata:
+  name: cluster
+spec:
+  localDiskSetup:
+    filesystems:
+    - device: /dev/md/nvmes
+      type: xfs
+    mounts:
+    - device: /dev/md/nvmes
+      mountPoint: /mnt/raid-disks/local
+      unsupportedOptions:
+      - prjquota
+    raids:
+    - name: nvmes
+      type: RAID0
+      RAID0:
+        devices:
+          modelRegex: nvme_card
+          nameRegex: ^/dev/nvme\d+n\d+$
```

Comment:
Why not use the stable names `/dev/disk/by-uuid/google-local-ssds-nvme-block/local-ssd-GENERATED_UUID1`? 
https://cloud.google.com/kubernetes-engine/docs/how-to/persistent-volumes/local-ssd#raw-block

Reply:
Current code only matches names from lsblk output. In theory it could be extended with matching files/symlinks from filesystem in backward compatible manner.

Reply:
@tnozicka your call then - I can send a PR extending the matching in node setup controller

Reply:
I think matching the official paths would be good but if we don't have it now, I don't want this PR to take the hit. thx